src/com/company/SortingAlgorithm.java

```diff
@@ -0,0 +1,27 @@
+package com.company;
+
+/*
+ * @param arr - user-entered array ;
+ * @param L (from) - start of array ;
```

Comment:
Param names don't match with actual names
@param L (from) - start of array ; - if if `start`, why it called `L/from/left` and not `start`?